x-pack/plugins/apm/server/routes/settings/agent_configuration.ts

```diff
@@ -262,6 +284,57 @@ const agentConfigurationAgentNameRoute = createApmServerRoute({
   },
 });
 
+// enables syncing with fleet-managed apm package policy
+const agentConfigurationFleetSync = createApmServerRoute({
+  endpoint: 'POST /api/apm/settings/agent-configuration/fleet_sync',
```

Comment:
Should we also add an API to stop the sync?

Reply:
If we want the button to also be a toggle, we can do that, otherwise the user is able to stop syncing by navigating to Stack management and delete the rule.